widgets/AvroDynamicPartitionedDataset-batchsink.json

```diff
@@ -2,6 +2,7 @@
   "metadata": {
     "spec-version": "1.3"
   },
+  "display-name": "Avro Dynamic Partitioned Dataset",
```

Comment:
If this is dataset, the other one should also be dataset?

Reply:
Not sure which is a better name. Their parent class is `PartitionedFileSetSink` but both of their names end with Dataset

Reply:
I think Dataset is better, its more well known